wsi/drm/drmdisplay.cpp

```diff
@@ -364,6 +364,24 @@ uint32_t DrmDisplay::FindPreferedDisplayMode(size_t modes_size) {
   return prefer_display_mode;
 }
 
+#ifdef KVM_HWC_PROPERTY
+uint32_t DrmDisplay::Find1080DisplayMode(size_t modes_size) {
+  uint32_t display_mode_1080p = 0;
+  if (modes_size >= prefer_display_mode_) {
+    int32_t p_width = 0, p_height = 0;
+    for (size_t i = 0; i < modes_size; i++) {
+      GetDisplayAttribute(i, HWCDisplayAttribute::kWidth, &p_width);
+      GetDisplayAttribute(i, HWCDisplayAttribute::kHeight, &p_height);
+      if (p_width <= 1920 && p_height <= 1080) {
```

Comment:
Shouldn't this rather be '==' than '<='? Otherwise it seems this loop would choose as example 640x480 happily as display_mode_1080p?

Reply:
Good point. Thanks to review this. I will update soon.